fe/fe-core/src/main/java/com/starrocks/sql/ShortCircuitPlanner.java

```diff
@@ -140,13 +167,27 @@ public Boolean visitLogicalProject(OptExpression optExpression, Void context) {
             return visitChild(optExpression, context);
         }
 
+        @Override
+        public Boolean visitLogicalLimit(OptExpression optExpression, Void context) {
+            if (optExpression.getOp().getOpType().equals(OperatorType.LOGICAL_LIMIT) &&
```

Comment:
the second condition is meaningless. Limit can only have one child.

Reply:
ok